.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -134,6 +138,7 @@
         This will include where each prebuilt is coming from.
       - Find an explanation for each change in each (online/offline) baseline.
         - If possible, figure out the cause and fix.
+          - See [/Documentation/servicing/common-prebuilt-issues.md](here) for docs on issues we see regularly.
```

Comment:
It would be good to at least include one in the initial checkin versus checking in an empty file.

Reply:
I've added the prebuilt issues we've seen in the last few months.